recurly/__init__.py

```diff
@@ -1738,6 +1738,35 @@ class MeasuredUnit(Resource):
         'updated_at',
     )
 
+class PercentageTier(Resource):
+
+    """Percentage tier associated to a set of tiers per currency 
+    in an add-on."""
+
+    nodename = 'tier'
+    inherits_currency = True
+
+    attributes = (
+        'ending_amount_in_cents',
+        'usage_percentage',
+    )
+
+class PercentageTierByCurrency(Resource):
```

Comment:
I went back and forth a bit with Faiber on the naming of this int he ruby library (IIRC) -- he ended up going with `CurrencyPercentageTiers` since it made it clear that it was a collection of tiers instead of a single tier.

Thoughts?  We should probably try to make the libraries consistent with each other for namign this node.

Reply:
Totally agree, this one you suggest is better

Reply:
And just remembering/correcting myself -- it'd be `CurrencyPercentageTier` since I forgot we have singular class names for thsi stuff.